flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/BuiltInMethods.scala

```diff
@@ -463,6 +463,9 @@ object BuiltInMethods {
   val SUBTRACT_MONTHS =
     Types.lookupMethod(classOf[DateTimeUtils], "subtractMonths", classOf[Long], classOf[Long])
 
+  val ADD_MONTHS_STRING_INT =
```

Comment:
Since there is already existing `subtractMonths` why could it be reused just with negative sign?